index.html

```diff
@@ -1134,10 +1134,10 @@ <h3 class="is-3 title" id="Help"><b>Help</b></h3>
 
       <div class="content has-text-centered">
         <div class="soc">
-          <a href="#" target="_blank"><i class="fab fa-github-alt fa-2x" role="img" aria-label="github" aria-hidden="true aria-hidden="true""></i><span class="visually-hidden">Link will open in new tab</span></a>
-          <a href="#" target="_blank"><i class="fab fa-youtube fa-2x" role="img" aria-label="youtube" aria-hidden="true aria-hidden="true""></i><span class="visually-hidden">Link will open in new tab</span></a>
-          <a href="#" target="_blank"><i class="fab fa-facebook fa-2x" role="img" aria-label="facebook" aria-hidden="true aria-hidden="true""></i><span class="visually-hidden">Link will open in new tab</span></a>
-          <a href="#" target="_blank"><i class="fab fa-twitter fa-2x" role="img" aria-label="twitter" aria-hidden="true aria-hidden="true""></i><span class="visually-hidden">Link will open in new tab</span></a>
+          <a href="#" target="_blank"><i class="fab fa-github-alt fa-2x" role="img" aria-label="github" aria-hidden="true"></i><span class="visually-hidden">Link will open in new tab</span></a>
```

Comment:
rol='img' - unnecessary since you use aria-hidden. But here you should notify users, about social media names